### PR TITLE
Add Marketing Analytics notebook

### PR DESCRIPTION
**Pull request by ivarpruijn**
Adds a Marketing Analytics notebook as Jupyter notebook and example notebook in the docs. Also removes any overlap with the product analytics notebook, and runs the latter notebook with doctests in the documentation.

### CODE REVIEW

File: notebooks/marketing-analytics.ipynb [hunk 1]
```diff
@@ -0,0 +1,1508 @@
+{
```
Comment (ivarpruijn) [inline notebooks/marketing-analytics.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    # TODO: don't understand why I can't use https://objectiv.io/docs/modeling/bach/api-reference/DataFrame/plot/
</pre><p>@vard @hendrik Any idea how to use our plot functionality here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/objectiv/objectiv-analytics/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (ivarpruijn) [inline notebooks/marketing-analytics.ipynb:1]: <p>@vard @hendrik Any idea how to do a plot here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/objectiv/objectiv-analytics/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (ivarpruijn) [inline notebooks/marketing-analytics.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    # TODO: filtering session_duration '0 days 00:00:00' provides different results, but shouldn't this already be handled by the <code>exclude_bounces</code> parameter?
</pre><p>@vard @hendrik Am I misunderstanding this functionality?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/objectiv/objectiv-analytics/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (ivarpruijn) [inline notebooks/marketing-analytics.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # TODO: Plat a graph of session duration per source; error: TypeError: value should be a 'Timedelta', 'NaT', or array of those. Got 'int' instead.
</pre><p>@vard @hendrik Any idea how to plot this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/objectiv/objectiv-analytics/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

**Comment by ivarpruijn** [comment]
@hendrik-obj @vard-obj Just added the updated product-analytics Jupyter notebook. Please let me know if you're happy with it, and then I'll proceed to update the version in the docs as well.

File: notebooks/basic-product-analytics.ipynb [hunk 2]
```diff
@@ -24,26 +21,16 @@
...
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "96aeea32",
-   "metadata": {},
-   "source": [
-    "### Import the required packages for this notebook\n",
-    "The open model hub package can be installed with `pip install objectiv-modelhub` (this installs Bach as well).  \n",
-    "If you are running this notebook from our quickstart, the model hub and Bach are already installed, so you don't have to install it separately."
```
Comment (hendrik-obj) [inline notebooks/basic-product-analytics.ipynb:46]: text changes throughout the notebook lgtm. only 1 thing: we once agreed to have the same text on top in all example notebooks (although already not all notebooks have that at this point). now this one is different. maybe @vincenthoogsteder has an opinion about changing these first cells (by doing that removing the `Import the required packages for this notebook` paragraph)?

Comment (vincenthoogsteder) [reply notebooks/basic-product-analytics.ipynb:46]: Sharp @hendrik-obj, I like this new intro in product analytics & marketing ones with the short intro & setup section quite a lot. Maybe we use that for the others as well to align them all again?

Comment (ivarpruijn) [reply notebooks/basic-product-analytics.ipynb:46]: My idea was to migrate all notebooks to this one-by-one as I make them all runnable in the docs. Already did this for three others I believe.

**Comment by vard-obj** [comment]
`product-analytics` notebook lgtm as well.

File: bach/docs/source/example-notebooks/marketing-analytics.rst [hunk 3]
```diff
@@ -0,0 +1,898 @@
...
+.. testsetup:: marketing-analytics
+	:skipif: engine is None
+
+	pd.set_option('display.max_colwidth', 93)
+
+.. doctest:: marketing-analytics
+	:skipif: engine is None
+
+	>>> # instantiate the model hub, and set the default time aggregation to daily
+	>>> from modelhub import ModelHub
```
Comment (hendrik-obj) [inline bach/docs/source/example-notebooks/marketing-analytics.rst:31]: do we need to add the other imports here as well 
```python
from bach import DataFrame
import pandas as pd
```

Comment (ivarpruijn) [reply bach/docs/source/example-notebooks/marketing-analytics.rst:31]: definitely, thanks, fixing

File: notebooks/marketing-analytics.ipynb [hunk 4]
```diff
@@ -0,0 +1,962 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from modelhub import ModelHub\n",
+    "from bach import DataFrame, display_sql_as_markdown\n",
+    "from datetime import datetime, timedelta\n",
```
Comment (vard-obj) [inline notebooks/marketing-analytics.ipynb:34]: Shall we also remove `display_sql_as_markdown`  and `from datetime import datetime, timedelta` imports  in the notebook as we don't use them anymore?

**Pull request merged**